### ISSUE TITLE
mup proxy logs [error 413] client intended to send too large body:

### ISSUE THREAD

**Issue by lc3t35** [zodern/meteor-up#950]
Mup version (`mup --version`): 1.4.4

Mup config 
``` 
"proxy": {
    "domains": "xxx",
    "clientUploadLimit": "0",
    "ssl": {
      "letsEncryptEmail": "xxx"
    }
  },
```

Output of command = mup proxy logs
```
[error] : *7 client intended to send too large body: 22676212 bytes, client: X.X.X.X, server: xxx, request: "POST /api/v1/medias HTTP/2.0", host: "xxx"
```

Despite I've configured clientUploadLimit, I can't upload some files (not excessive size here 22.6MB).
How can check that nginx config has client_max_body_size set to 0 ?
`mup proxy nginx-config` does not show such prop ...

**Comment by lc3t35** [comment]
I confirm that this param has no effect on mup-nginx-proxy configuration (client_max_body_size 10M), and if you change manually the file /opt/mup-nginx-proxy/config/nginx-default.conf
````
# Default shared nginx config
client_max_body_size 200M;
````
then restart the mup-nginx-proxy docker, it solved the 413 error.

Can someone check was is missing in mup's proxy setup ?

**Comment by zodern** [comment]
This is fixed in version 1.4.5.

**Comment by EloyID** [comment]
Hi! I'm using the version 1.5.1 of mup, and I'm having still this issue.

I get the error 413 Request Entity Too Large from Nginx

```
 "proxy": {
    "domains": "xxx.xxx.com",
    "clientUploadLimit": "50M", //also tried "0"
    "ssl": {
      "letsEncryptEmail": "email@email.com",
      "forceSSL": true
    }
  }
```

Any news on how to solve this ?
